automation/deploy_hco.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
+
+set -ex
+
+HCO_VERSION=1.4.0
+CMD=oc
+NS=kubevirt-hyperconverged
+
+# Create a catalog source
+cat <<EOF | ${CMD} apply -f -
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: hco-unstable-catalog-source
```

Comment:
according to the value of `${HCO_VERSION}`, this does not seem to be the unstable version.

Reply:
fixed